nat-lab/tests/mesh_api.py

```diff
@@ -248,7 +291,7 @@ def generate_peer_config(node: Node, allowed_ips: str) -> str:
         wg_conf = (
             f"[Interface]\nPrivateKey = {server_config['private_key']}\nListenPort ="
             f" {server_config['port']}\nAddress = 100.64.0.1/10,"
-            " fc74:656c:696f::1/64\n\n"
+            f" {LIBTELIO_IPV6_WG_SUBNET}::1/64\n\n"
```

Comment:
:thinking: I am thinking whether it really makes sense to add IPv6 address here, as in reality NordVPN servers do not have in-tunnel IPv6 address (we reserved one, but at this moment, those addresses are not actually applied on the servers. So maybe until this time, we should not have it in nat-lab VPN servers too?

Reply:
My thinking was the same, but than I bumped into failing tests, because of this. I do not remember which tests exactly, but I will try to mimic that now ...

Reply:
Fixed